src/main/java/duke/task/ToDo.java

```diff
@@ -0,0 +1,25 @@
+package duke.task;
+
+/** A To-Do task. */
```

Comment:
Let's try to standardise the format of your Javadoc comments:
```
/**
 * Returns lateral location of the specified position.
 * If the position is unset, NaN is returned.
 *
 * @param x  X coordinate of position.
 * @param y Y coordinate of position.
 * @param zone Zone of position.
 * @return Lateral location.
 * @throws IllegalArgumentException  If zone is <= 0.
 */
public double computeLocation(double x, double y, int zone)
    throws IllegalArgumentException {
  ...
}
```

So for example, `/**` should be [on a separate line](https://nus-cs2103-ay1920s2.github.io/website/coding-standards/java/intermediate.html#layout).